dev/tools/update_icons.dart

```diff
@@ -352,65 +362,57 @@ class _Icon {
 
     if (id.endsWith('_outlined') && id!='insert_chart_outlined') {
       style = IconStyle.outlined;
-      shortId = id.replaceAll('_outlined', '');
+      shortId = id.replaceFirst('_outlined', '');
     } else if (id.endsWith('_rounded')) {
       style = IconStyle.rounded;
-      shortId = id.replaceAll('_rounded', '');
+      shortId = id.replaceFirst('_rounded', '');
     } else if (id.endsWith('_sharp')) {
       style = IconStyle.sharp;
-      shortId = id.replaceAll('_sharp', '');
+      shortId = id.replaceFirst('_sharp', '');
```

Comment:
The "replaceFirst" change is an improvement for sure, but it would still get it wrong e.g. if we have something like "foo_outlined_bar_outlined". Can we just do a substring so that we are definitely stripping the end?

Reply:
Done